### PR TITLE
@ #164 | should update nodejs dev guide with support for nodejs debugging

### PR DESCRIPTION
**Pull request by hoatle**
connect #164 

- need to review technical content (@hieptranquoc)
- need to review English content (@hoavt)

### CODE REVIEW

**Review by phuonglm — DISMISSED** [review]

File: docs/nodejs_dev_guide.rst
Comment (hoavt) [inline docs/nodejs_dev_guide.rst]: there are code changes, debugging, etc.

File: docs/nodejs_dev_guide.rst
Comment (hoavt) [inline docs/nodejs_dev_guide.rst]: should use ``vagrant ssh`` for clearer

File: docs/nodejs_dev_guide.rst
Comment (hoavt) [inline docs/nodejs_dev_guide.rst]: two-way sync for node_modules

File: docs/nodejs_dev_guide.rst
Comment (hoavt) [inline docs/nodejs_dev_guide.rst]: To have 2-way sync,

File: docs/nodejs_dev_guide.rst
Comment (hoavt) [inline docs/nodejs_dev_guide.rst]: Create the ``node_modules`` directory

File: docs/nodejs_dev_guide.rst
Comment (hoavt) [inline docs/nodejs_dev_guide.rst]: Add ``gitignore`` for the ``node_modules`` content, except ``.gitkeep`` to keep the empty directory. This is required
by the configuration below.

Comment (hoavt) [inline docs/nodejs_dev_guide.rst]: The configuration below is "vagrant_config_override.json"?

File: docs/nodejs_dev_guide.rst
Comment (hoavt) [inline docs/nodejs_dev_guide.rst]: Configure the ``vagrant_config_override.json`` file with the following content:

File: docs/nodejs_dev_guide.rst
Comment (hoavt) [inline docs/nodejs_dev_guide.rst]: to get it to install the ``npm`` packages again:

File: docs/nodejs_dev_guide.rst
Comment (hoavt) [inline docs/nodejs_dev_guide.rst]: ``npm``

File: docs/nodejs_dev_guide.rst
Comment (hoavt) [inline docs/nodejs_dev_guide.rst]: Create the...

File: docs/nodejs_dev_guide.rst
Comment (hoavt) [inline docs/nodejs_dev_guide.rst]: Create the...

File: docs/nodejs_dev_guide.rst
Comment (hoavt) [inline docs/nodejs_dev_guide.rst]: should use ``vagrant ssh`` for be clearer

File: docs/nodejs_dev_guide.rst
Comment (hoavt) [inline docs/nodejs_dev_guide.rst]: dev mode or prod mode?

File: docs/nodejs_dev_guide.rst
Comment (hoavt) [inline docs/nodejs_dev_guide.rst]: ``vagrant ssh``

File: docs/nodejs_dev_guide.rst
Comment (hoavt) [inline docs/nodejs_dev_guide.rst]: To have the most up-to-date  and more information, please checkout the ``README.md`` file....

File: docs/nodejs_dev_guide.rst
Comment (hoavt) [inline docs/nodejs_dev_guide.rst]: This is --> These are

**Review by hoavt — COMMENTED** [review]
@hoatle: Please check my comments

**Comment by hoatle** [comment]
@phuonglm don't approve (blindly?) like that, you should approve only after @hieptranquoc and @hoavt's reviewing.

**Comment by hoatle** [comment]
@hoavt feedbacks applied

**Comment by hoatle** [comment]
@hieptranquoc need your check on the technical content is there is any problem.

File: docs/nodejs_dev_guide.rst
Comment (hoavt) [inline docs/nodejs_dev_guide.rst]: --> To get the most....

**Review by hoavt — COMMENTED** [review]
@hoatle: Please check my comments

**Comment by hieptranquoc** [comment]
@hoatle reviewed, technical content is ok

**Comment by hoavt** [comment]
It's ok to merge.

**Comment by hoatle** [comment]
@hoavt please approve.

**Review by phuonglm — APPROVED** [review]

**Review by hoavt — APPROVED** [review]

**Pull request merged**
